src/components/LiquidityChartRangeInput/index.tsx

```diff
@@ -157,7 +157,7 @@ export default function LiquidityChartRangeInput({
   )
 
   if (error) {
-    sendEvent('exception', { description: error.toString(), fatal: false })
+    Sentry.captureMessage(error.toString(), 'log')
```

Comment:
Is there a reason this (and other usages) can't import only the used bit, eg `import { captureMessage } from '@sentry/react`?

Reply:
No, I actually thought about it, but somewhat found `captureMessage` alone a bit unintuitive. Not a strong preference tbh.

Reply:
Following up on this, we have a choice between:
```
import * as Sentry from 'sentry'
```
vs
```
import {init as initSentry} from 'sentry'
```
while I am a huge proponent of latter approach in code in general (over defaults and import all), I think former makes more sense in this context.